lms/templates/instructor/instructor_dashboard_2/metrics.html

```diff
@@ -66,7 +66,6 @@ <h3 class="hd hd-3">${_("Grade Distribution Data")}</h3>
   %endfor
   <script>
     $(function () {
-      var firstLoad = true;
```

Comment:
The purpose of this was to prevent the metrics from re-generating on subsequent visits to the tab from within the Instructor Dash. This was for performance reasons.

Now it re-generates on each visit to the tab.